modules/internal/ChapelArray.chpl

```diff
@@ -1523,8 +1531,8 @@ module ChapelArray {
 
     // associative array interface
     /* Yield the domain indices in sorted order */
-    iter sorted() {
-      for i in _value.dsiSorted() {
+    iter sorted(comparator:?t = chpl_defaultComparator()) {
```

Comment:
I don't think that works the way I would expect it to. In particular, if I have two files, library.chpl and test.chpl:

library.chpl:

    var x = 1;

test.chpl:

    writeln(library.x);

now if I just do:
     chpl test.chpl

I get the error:

    test.chpl:1: error: 'library' undeclared (first use this function)

So, if Sort.defaultComparator worked in this case, it would be due to some special relationship that Sort has (such as being used in other places) and that seems iffy to me.